packages/dd-trace/src/config.js

```diff
@@ -902,6 +918,9 @@ class Config {
     }
     this._setString(opts, 'iast.telemetryVerbosity', options.iast && options.iast.telemetryVerbosity)
     this._setBoolean(opts, 'isCiVisibility', options.isCiVisibility)
+    this._setBoolean(opts, 'llmobs.agentlessEnabled', options.llmobs?.agentlessEnabled)
```

Comment:
```suggestion
    this._setBoolean(opts, 'llmobs.agentlessEnabled', options.llmobs?.agentlessEnabled)
    this._setBoolean(opts, 'llmobs.enabled', options.llmobs)
```

Reply:
with regards to my comment below (& updated the pr description too for clarity) - could i maybe do something like:

```javascript
this._setBoolean(opts, 'llmobs.enabled', this._env['llmobs.enabled'] ?? options.llmobs)
```

since it looks like although `options` take precedence over `environment`, `environment` is calculated _before_ `options`, so I can use `this._env` in computing `this._opts['llmobs.enabled']`. wdyt?

Reply:
this way, i can effectively have the environment variable take precedence over the options and respect the general flow of the config structure

Reply:
i think resolved using `_applyCalculated` in https://github.com/DataDog/dd-trace-js/pull/4696/commits/8fd4912c4c710799c8d12ee5a47a975a3b44a799